src/backend/distributed/executor/multi_utility.c

```diff
@@ -384,6 +375,11 @@ multi_ProcessUtility(Node *parsetree,
 		SetUserIdAndSecContext(savedUserId, savedSecurityContext);
 	}
 
+	if (ddlJob != NULL)
+	{
+		ExecuteDistributedDDLJob(ddlJob);
+	}
+
 	/* we run VacuumStmt after standard hook to benefit from its checks and locking */
 	if (IsA(parsetree, VacuumStmt))
```

Comment:
I assume vacuum can't be done via ExecuteDistributedDDLJob due to transactional behaviour and such?